server/core/src/rodsServer.cpp

```diff
@@ -673,6 +796,7 @@ int serverMain(
 
             }
             ix::cron::cron::get()->run();
+            // TODO Consider letting the CRON manager handle the logging of stacktrace files.
```

Comment:
review the CRON manager code to see if there are benefits to using it

Reply:
- single loop, no watcher to watch the watcher required.
- easier to reason about later.

Reply:
if many stacktrace files exist on a server, processing them here will block incoming client requests from being serviced.

Reply:
true... how many are we planning on having?   i mean, maybe stopping and processing broken things is ... good ... at that point?

Reply:
hopefully none :) -- however, one file means we are touching the disk. every time we do there here, we block for the amount of work the cron manager has to handle.

i need to review the cron code tho. i can't remember how it actually works.

Reply:
+1 for eventually using the cron manager, assuming everything checks out

Reply:
done. needs to be tested.